Improvements on  current Chinese translation

stevenzhu25 opened the pull request:
Since the publishing of the initial translation, I feel that some part of the content is not really that fluent to native Chinese speakers (Some of those looks awkward), hence i'm trying to improve the currently translated pages (update those that aren't up to date) and translate more pages.

For Simplified Chinese speakers: If you would like to help me, just send me an email or comment on this PR, i'll be happy to add you to that repo. Proofreading is strongly welcomed (and wanted)
如果您愿意一起参加对网站的翻译， 欢迎在本申请下方评论或给我写邮件（我的邮箱可在Profile页查看）。 也欢迎对翻译的不通顺的语句提出改进。

Current Staging Website Link: https://le-website-chn-draft.stevenz.net/
当前翻译版本链接（非官方）：https://le-website-chn-draft.stevenz.net/

谢谢
Thank you

tdelmas commented on content/zh-cn/about.md:
```diff
@@ -10,15 +10,15 @@ menu:
 ---
 
-Let's Encrypt是一个免费的，自动化的，开放的证书颁发机构（CA），为公众的利益而运行。 它是由[Internet Security Research Group（ISRG）]（/ isrg /）提供的服务。
+Let's Encrypt是一个免费的，自动化的，开放的证书颁发机构（CA），为公众的利益而运行。 它是由[Internet Security Research Group（ISRG）]（/isrg/）提供的服务。
```
```suggestion
Let's Encrypt是一个免费的，自动化的，开放的证书颁发机构（CA），为公众的利益而运行。 它是由[Internet Security Research Group（ISRG）](https://www.abetterinternet.org/）提供的服务。
```

tdelmas commented:
Thank you for that work!
Could you use that PR to also fix https://github.com/letsencrypt/website/blob/master/i18n/zh-cn.toml#L33

```
[footer_policies]
other = """
View our <a href="/zh-cn/privacy/">隐私政策</a>.<br>
View our <a href="/zh-cn/trademarks/">商标政策</a>.
"""
```
To translate "View our"?

stevenzhu25 commented:
> 
> 
> Thank you for that work!
> Could you use that PR to also fix https://github.com/letsencrypt/website/blob/master/i18n/zh-cn.toml#L33
> 
> ```
> [footer_policies]
> other = """
> View our <a href="/zh-cn/privacy/">隐私政策</a>.<br>
> View our <a href="/zh-cn/trademarks/">商标政策</a>.
> """
> ```
> 
> To translate "View our"?

Fixed. Also added the missing 3 sentences 😄

tdelmas commented:
Thanks!

stevenzhu25 commented:
Checklist:

- Improvements Needed: 
-- getting-started.md

- Translation Needed:
-- All docs pages
-- how-it-works.md
-- nondiscrimination.md
-- privacy.md
-- stats-dashboard.md
-- stats.md
-- thankyou.md
-- trademarks.md
-- upcoming-features.md

I'll come back and finish as much as I could... after dinner.

stevenzhu25 commented:
@tdelmas can you check on this when available? I'm not sure why Travis-CI keeps failing. (It's referring to something on i18n page?). Also, this PR is ready to merge! (I'm currently not going to touch pages related to legal and glossary... For now)

Thanks.

tdelmas commented:
Sorry I missed your message! I should have time this weekend to check and merge it.

A7T commented:
@stevenzhu25 您好，我的母语是中文，我希望参加中文文档的校对、润色工作。
我刚接触Git，对工作流程不太熟悉，如果我添了麻烦，请不吝批评。

stevenzhu25 commented:
@A7T 欢迎欢迎 已经邀请您了。😀

tdelmas commented on content/zh-cn/about.md:
```diff
@@ -10,17 +10,18 @@ menu:
 
-我们有一个页面，其中包含更多关于[Let's Encrypt CA是如何工作的]({{< ref "/how-it-works.md" >}})的详细信息
+
+请访问[此页面](/how-it-works/)获取更多关于Let's Encrypt CA工作方式的详细信息
```
Please keep the `ref` macro

```suggestion
请访问[此页面]({{< ref "/how-it-works.md" >}})获取更多关于Let's Encrypt CA工作方式的详细信息
```

tdelmas commented on content/zh-cn/docs/integration-guide.md:
```diff
@@ -1,4 +1,129 @@
+
+我们将始终尽可能提前通知此类更改，但如果在某些组件中发现严重的安全漏洞，我们可能需要在短期内或立即进行更改。 特别是对于中级证书改变，你不应该将中级证书写入代码，而应该使用[`Link: rel="up"`](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.3.1)来自ACME协议的标头（Header），因为中级证书可能会改变。
+
+同样，我们可能会在更新时更改服务条款（ToS）的链接。 避免硬编码ToS URL，而是依赖[`Link：rel =“服务条款”`](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2)标头（Header）确定要使用的服务条款链接。
```
`rel="terms-of-service"` should not be translated

```suggestion
同样，我们可能会在更新时更改服务条款（ToS）的链接。 避免硬编码ToS URL，而是依赖[`Link：rel =“terms-of-service”`](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2)标头（Header）确定要使用的服务条款链接。
```

tdelmas commented:
Thank you very much for all that work.

Do you prefer I merge it now or wait a review from @shulongdeng or @A7T  ? 

(from a technical point of view I think it's ready to be merged)

shulongdeng commented:
For `how-it-works.md`, it's quite legit from cryptography point of view.
The other documents from @stevenzhu25  seem quite good too.
